### ISSUE TITLE
Create virtual in typegoose

### ISSUE THREAD

**Issue by vasyl97kharchenko** [typegoose/typegoose#899]
Hello, I need your help. How do I transform this in mongoose to typegoose so that when I do entity.populate('subtotal') -> I get subtotal populated value?
```
schema.virtual('subtotal').get(function subtotal() {
    return this.total - this.tax - this.shipping;
});
```
Is it the only way to do so?
class Entity {
}

**Comment by hasezoey** [comment]
> Is it the only way to do so?

yes, class `get & set` directly map to mongoose's `virtual.get & set`

there is also the [Virtuals](https://typegoose.github.io/typegoose/docs/api/virtuals/) documentation

---

aside from that `entity.populate('subtotal')` does not look valid to me (`subtotal` is not a populate-able path in both examples)
